Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -175,15 +175,20 @@ def bump_cask_pr
     run_cask_style(cask, old_contents, args: args)
 
     branch_name = "bump-#{cask.token}"
-    commit_message = "Update #{cask.token}"
-    if new_version.present?
-      if new_version.before_comma != old_version.before_comma
-        new_version = new_version.before_comma
-        old_version = old_version.before_comma
-      end
+    commit_message = if new_version.present?
       branch_name += "-#{new_version.tr(",:", "-")}"
-      commit_message += " from #{old_version} to #{new_version}"
+
+      new_version = new_version.before_comma if new_version.before_comma != old_version.before_comma
+
+      "#{cask.token} #{new_version}"
+    elsif new_base_url.present?
+      "#{cask.token}: update URL"
+    elsif new_hash.present?
+      "#{cask.token}: update checksum"
+    else
+      raise
```

Comment:
shouldn't we just set a default for this?

Reply:
Agreed, and if we need to `raise`: present an error message here.

Reply:
This is unreachable since we already check above that one of these arguments is provided.

Reply:
I guess it is technically reachable if you only set one of these and it's empty, since the check above uses `.nil?`.

Reply:
Then why include the block?

Reply:
Refactored this a bit to handle empty arguments earlier so this isn't needed anymore.